script/fetch-issues.js

```diff
@@ -141,22 +159,76 @@ async function fetchNextPage(owner, name, isIssue, data, cursor, pageCount) {
         }
       };
       if (!isIssue) {
+        let raw_comments, comments;
+        // collect review comments
+        raw_comments = node.reviews.nodes.map(node => node.comments.nodes);
+        raw_comments = [].concat.apply([], raw_comments);
+        // collect issue comments
+        raw_comments = raw_comments.concat(node.comments.nodes);
+        // filter useless reviews
+        raw_comments = raw_comments.filter(node => {
+          let flag = true;
+          // filter comments that don't need meta-reviews
+          for (const ignore_content of ignore_review_content.split(' ')) {
```

Comment:
Why not `Array.forEach` ?

... or `Array.reduce`

Reply:
I see you've used `for` loop before in previous PR.

I'm not sure what the project/original author prefers, but need to keep everything consistent.

Reply:
I guess just leave it as is, if you're not going to use `reduce`